.github/workflows/arm64.yml

```diff
@@ -15,19 +15,78 @@ env:
+          aws-region: eu-west-2
+      - name: "Start EC2 runner"
+        id: start-ec2-runner
+        uses: machulav/ec2-github-runner@4e0303de215db88e1c489e07a15ca4d867f488ea
+        with:
+          mode: start
+          github-token: ${{ secrets.BOT_GITHUB_TOKEN }}
+          ec2-image-id: ami-0ec31c3e3b729bb6d
+          ec2-instance-type: c6g.2xlarge
+          subnet-id: subnet-0777c70a4cd9ce944
+          security-group-id: sg-0f89a325d9eb147cb
+          aws-resource-tags: >
+            [
+              {"Key": "Name", "Value": "arm64-acs-packaging-gh-runner"},
+              {"Key": "GitHubRepository", "Value": "${{ github.repository }}"}
+            ]
+
   arm64_health_check:
-    name: ARM64 Health Check
-    runs-on: [self-hosted, linux, arm64]
+    name: "ARM64 Health Check"
+    needs: start-runner
+    runs-on: ${{ needs.start-runner.outputs.label }}
     outputs:
       test_failure: ${{ steps.persist_test_failure_flag.outputs.test_failure }}
     steps:
       - uses: actions/checkout@v3
         with:
           persist-credentials: false
           fetch-depth: 0
-      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.36.0
-      - uses: Alfresco/alfresco-build-tools/.github/actions/free-hosted-runner-disk-space@v1.36.0
-      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.36.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v3.6.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/free-hosted-runner-disk-space@v3.6.0
+      - name: "Install required software"
+        run: |
+          # Install Docker as per https://docs.docker.com/engine/install/ubuntu/
+          sudo apt-get update
+          sudo apt-get install ca-certificates curl gnupg -y
+          sudo install -m 0755 -d /etc/apt/keyrings
+          curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+          sudo chmod a+r /etc/apt/keyrings/docker.gpg
+          echo \
+            "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+            "$(. /etc/os-release && echo "$VERSION_CODENAME")" stable" | \
+            sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+          sudo apt-get update
+          # Additionally install git, zip and unzip (required by SDKMAN to install Maven), python3
+          sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin git zip unzip python3 python3-pip -y
+          # Allow usage of "docker-compose" rather than "docker compose" in bash scripts
+          ln -f -s /usr/libexec/docker/cli-plugins/docker-compose /usr/local/bin/docker-compose
+          # Workaround due to $HOME not being set causing issues with settings.xml installation in setup-build-java
+          mkdir -p /root/.m2
```

Comment:
you are using sudo in the previous commands but if this is not failing here without sudo it means that current user is root so you don't need sudo in any of the previous commands

Reply:
yeah, the `sudo`s here are redundant as the `root` user is the only user that can be engaged by this action. I'll get rid of the unnecessary `sudo`s 👍